pkg/proto/sqlflow.proto

```diff
@@ -34,44 +34,39 @@ service SQLFlow {
     // Fetch fetches the SQLFlow job phase and logs in a polling manner. A corresponding
     // client can be implemented as
     //
-    // job = c.run(request).job
-    // token = fetch_token(job=job)
-    // while True:
-    //     r = c.fetch(token)
-    //     print(r.logs.content)
-    //     if r.phase == SUCCEEDED/FAILED/ERROR and r.logs.no_more_log:
-    //         break
-    //     token = r.token
-    rpc Fetch (FetchToken) returns (FetchResponse);
+    // wfJob := Submit(argoYAML)
+    // req := newFetchRequest(wfJob)
+    // for {
+    //   res := Fetch(req)
+    //   fmt.Println(res.Logs)
+    //   if isComplete(res) {
+    //     break
+    //   }
+    //   req = res.NewFetchRequest
+    //   time.Sleep(time.Second)
+    // }
+    //
+    rpc Fetch (FetchRequest) returns (FetchResponse);
 }
 
 message Job {
     string id = 1;
 }
 
-message FetchToken {
+message FetchRequest {
     Job job = 1;
     // the following fields keep the fetching state
     string step_id = 2;
     string log_offset = 3;
-    bool no_more_log = 4;  // True if no more logs
+    bool finish_fetching = 4;  // True if no more logs
```

Comment:
I prefer the word EOF here

Reply:
`finish_fetching` should be under `FetchResponse` but not in `Request`? We do not need to send `finish_fetching` to the server I think

Reply:
You're right, move `finish_fetching` to `FetchResponse`.